lib/ui/semantics.dart

```diff
@@ -201,6 +207,50 @@ class SemanticsAction {
   /// movement should extend (or start) a selection.
   static const SemanticsAction moveCursorBackwardByWord = SemanticsAction._(_kMoveCursorBackwardByWordIndex, 'moveCursorBackwardByWord');
 
+  /// Move the input focus to the respective widget.
+  ///
+  /// Most commonly, the input focus determines what widget will receive
```

Comment:
```suggestion
  /// Most commonly, the input focus determines which widget will receive
```

Reply:
Oops! I slapped on the autosubmit label without realizing that my `git push` failed 🤦  I'll send the doc fix in a separate PR.